docs/plugins/repository-azure.asciidoc

```diff
@@ -121,8 +121,8 @@ The Azure repository supports following settings:
 `chunk_size`::
 
     Big files can be broken down into chunks during snapshotting if needed.
-    The chunk size can be specified in bytes or by using size value notation,
-    i.e. `1g`, `10m`, `5k`. Defaults to `64m` (64m max)
+    The chunk size can be specified by using size value notation,
+    i.e. `10MB`, `5KB`, `500B`. Defaults to `64MB` (64MB max)
```

Comment:
While you're at it, I'd flip it around to get rid of the passive construction and get rid of the "i.e." I'd probably also be explicit instead of using "size value notation". Something like:

> Specify the chunk size as a value and unit, for example: `10MB`, `5KB`, `500B`. Defaults to `64MB` (64MB max).

Reply:
Changes have been made per your advice.